Sources/KeyboardShortcuts/KeyboardShortcuts.swift

```diff
@@ -270,6 +263,30 @@ public enum KeyboardShortcuts {
 		reset(names)
 	}
 
+	/**
+	Reset all keyboard shortcuts and remove their stored values from `UserDefaults`.
+
+	This method resets all keyboard shortcuts registered via `KeyboardShortcuts` and removes any associated data stored in `UserDefaults`.
+
+	Use this method to completely reset the state of all keyboard shortcuts in your app, including removing any saved user-defined shortcuts and default shortcuts.
+
+	- Note: This action cannot be undone. All shortcuts will be removed, including any default shortcuts defined in `Name`. If you want to reset shortcuts back to their default values, consider using `.reset(_:)` instead.
```

Comment:
> If you want to reset shortcuts back to their default values, consider using `.reset(_:)` instead.

I don't think it makes sense that this has different behavior than `.reset()`. `resetAll` should just be `reset` for **all**.